lascli/parser/models.py

```diff
@@ -71,6 +74,11 @@ def list_data_bundles(las_client: Client, model_id, max_results, next_token):
     return las_client.list_data_bundles(model_id, max_results=max_results, next_token=next_token)
 
 
+def list_all_data_bundles(las_client: Client, max_results, next_token):
+    for model in list_models(las_client=las_client, max_results=max_results, next_token=next_token)['models']:
+        return las_client.list_data_bundles(model['modelId'], max_results=max_results, next_token=next_token)
```

Comment:
Is it intended that we are returning on first iteration? Maybe `yield from` was intended here?